src/ObjectUtils/SymbolsFileTest.cpp

```diff
@@ -71,4 +71,46 @@ TEST(SymbolsFile, FailToCreateSymbolsFile) {
   EXPECT_THAT(invalid_file, HasError("File does not exist"));
 }
 
+TEST(GetAllowedSymbolFilenamesForModule, ElfFile) {
+  orbit_grpc_protos::ModuleInfo::ObjectFileType object_file_type =
+      orbit_grpc_protos::ModuleInfo::kElfFile;
```

Comment:
Nit: This can be a constant `constexpr ... kObjectFileType`

Reply:
done